k3os/install.sh

```diff
@@ -172,28 +171,31 @@ do_copy()
     mount --bind /etc etc
     mount -r --rbind /lib lib
     mount -r --rbind /sys sys
-    chroot . /bin/bash <<"EOF"
-    # invoke k3s to set up data dir
-    k3s agent --no-flannel &>/dev/null || true
-    # start containerd
-    /var/lib/rancher/k3s/data/current/bin/containerd \
-    -c /var/lib/rancher/k3s/agent/etc/containerd/config.toml \
-    -a /run/k3s/containerd/containerd.sock \
-    --state /run/k3s/containerd \
-    --root /var/lib/rancher/k3s/agent/containerd &>/dev/null &
-
-    #wait for containerd to be ready
-    until ctr --connect-timeout 1s version>/dev/null
-    do
-      sleep 1
-    done
-    # import images
-    ctr -n k8s.io images import /var/lib/rancher/k3s/agent/images/harvester*
-    rm /var/lib/rancher/k3s/agent/images/harvester*
-    # stop containerd
-    pkill containerd
-    exit
+    if [ -f "${root_path}/${offline_image_path}" ]; then
```

Comment:
The whole image handling(including the mount and unmount) can be wrapped in the if statement in L160